app/controllers/host_aggregate_controller.rb

```diff
@@ -130,19 +109,6 @@ def update
     end
   end
 
-  def update_finished
```

Comment:
Where is this function referenced, have you remove that too?

Reply:
Already removed all uses of update_finished in this file and it doesnt seem to be used outside of this file.